build fails - Development additions not found (missing dev-additions.js)

I tried to debug on my own for #193 but build fails (on MacOS, but I guess on others as well) both for `npm run dev` and `npm build-main` See details for each below.

- `npm run dev`
Basically it says "Development additions not found" which is due to missing `./dev-additions.js`
> https://github.com/esteemapp/esteem-surfer/blob/4f1c4d567d02ab357430ba2e8420895e7f96953b/app/main.dev.js#L143-L148

```
$ npm run dev

> esteem-surfer@2.0.5 dev /Users/eugene/Documents/GitHub/esteem-surfer
> cross-env START_HOT=1 node -r babel-register ./internals/scripts/CheckPortInUse.js && cross-env START_HOT=1 yarn start-renderer-dev

yarn run v1.9.2
$ cross-env NODE_ENV=development node --trace-warnings -r babel-register ./node_modules/webpack-dev-server/bin/webpack-dev-server --config webpack.config.renderer.dev.js
Starting Main Process...

> esteem-surfer@2.0.5 start-main-dev /Users/eugene/Documents/GitHub/esteem-surfer
> cross-env HOT=1 NODE_ENV=development electron -r babel-register ./app/main.dev.js

Development additions not found
[93634:0129/203959.792819:ERROR:CONSOLE(7574)] "Extension server error: Object not found: <top>", source: chrome-devtools://devtools/bundled/inspector.js (7574)
✖ ｢wdm｣:
ERROR in ./app/backend/esteem-client.js
Module not found: Error: Can't resolve '../config' in '/Users/eugene/Documents/GitHub/esteem-surfer/app/backend'
 @ ./app/backend/esteem-client.js 12:14-34
 @ ./app/actions/search-results.js
 @ ./app/store/configureStore.dev.js
 @ ./app/store/configureStore.js
 @ ./app/index.js
 @ multi (webpack)-dev-server/client?http://localhost:1212 (webpack)/hot/dev-server.js react-hot-loader/patch webpack-dev-server/client?http://localhost:1212/ webpack/hot/only-dev-server ./app/index.js

ERROR in ./app/containers/App.js
Module not found: Error: Can't resolve '../config' in '/Users/eugene/Documents/GitHub/esteem-surfer/app/containers'
 @ ./app/containers/App.js 83:14-34
 @ ./app/Routes.js
 @ ./app/containers/Root.js
 @ ./app/index.js
 @ multi (webpack)-dev-server/client?http://localhost:1212 (webpack)/hot/dev-server.js react-hot-loader/patch webpack-dev-server/client?http://localhost:1212/ webpack/hot/only-dev-server ./app/index.js
Not rewriting GET /dist/renderer.dev.js because the path includes a dot (.) character.
```

Anyway, it still runs esteem-surfer but with errors:
![](https://user-images.githubusercontent.com/38183982/51939250-e90f2b80-2406-11e9-9e0b-617c4abc1d0a.png)


- `yarn build-main` (which is also required by `npm run test`)
```
WARNING in ./app/main.dev.js
Module not found: Error: Can't resolve './dev-additions.js' in '/Users/eugene/Documents/GitHub/esteem-surfer/app'
 @ ./app/main.dev.js
```

this is just a warning, you should be able to build without it as well. issue lies on `config.js` file... `Module not found: Error: Can't resolve '../config' in '/Users/eugene/Documents/GitHub/esteem-surfer/app/backend'` which you should create from example file. These holds our backend info, some services might not work without them but you can still fill them in as a blank and get build up and running.

--- a/README.md
+++ b/README.md
@@ -51,6 +51,14 @@ $ cd esteem-surfer
 $ yarn
 ```
 
+### Create a dummy config.js
+
+This is for the backend server configuration. Actual server addresses are not needed for most functions.
+
+```
+$ copy app/config.example.js app/config.js
+```
+
 ### Run in development mode
 
 ```

--- a/README.md
+++ b/README.md
@@ -56,7 +56,7 @@ $ yarn
 This is for the backend server configuration. Actual server addresses are not needed for most functions.
 
 ```
-$ copy app/config.example.js app/config.js
+$ cp app/config.example.js app/config.js
 ```
 
 ### Run in development mode

